### PR TITLE
v1.15.0 release

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+
+3. The `nervesproject/nerves_system_br` Docker image that corresponds to this
+   release has been updated from Ubunto 18.04 LTS to Ubuntu 20.04 LTS. This
+   fixed an internal compiler error that was seen, but resulted in some package
+   changes (no more Python 2, for example) that might affect scripts using it.
+
+* Improvements
+  * It's not possible to include executable files for other CPU architectures in
+    the root filesystem without have the scrubber find them and report an error.
+    To use this feature, add the file `.noscrubber` to the directory with the files.
```
Comment (ConnorRigby) [inline CHANGELOG.md:37]: the file is called `.noscrub`

**Review by ConnorRigby — APPROVED** [review]
One minor typo, otherwise looks great!

File: CHANGELOG.md (hunk 1 above)
Comment (fhunleth) [reply CHANGELOG.md]: Fixed. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+   5b95a5d](https://git.busybox.net/buildroot/commit/?id=5b95a5dc27c0d8002c00bda1c867ddea9218087e)
+   for more info.
+
+3. The `nervesproject/nerves_system_br` Docker image that corresponds to this
+   release has been updated from Ubunto 18.04 LTS to Ubuntu 20.04 LTS. This
+   fixed an internal compiler error that was seen, but resulted in some package
+   changes (no more Python 2, for example) that might affect scripts using it.
+
+* Improvements
+  * It's not possible to include executable files for other CPU architectures in
```
Comment (jjcarstens) [inline CHANGELOG.md:35]:
```suggestion
  * It's now possible to include executable files for other CPU architectures in
```

Comment (fhunleth) [reply CHANGELOG.md:35]: Thanks! Somehow I missed your comment, but I fixed it in `main`.

**Pull request merged**